todo_app/data/trello_items.py

```diff
@@ -0,0 +1,58 @@
+import os
+import requests
+
+from todo_app.data.item import Item
+
+def get_items():
+    reqUrl = "https://api.trello.com/1/boards/65afd1d3a662c6c7228ce261/lists"
+
+    query_parameters =  {
+        "key": os.getenv("TRELLO_API_KEY"),
+        "token": os.getenv("TRELLO_API_TOKEN"),
+        "cards": "open",
+        "card_fields":"id,name"
+    }
+
+
+    response = requests.get(reqUrl, params= query_parameters)
+
+    response_json = response.json()
+
+    cards = []
+    for trello_list in response_json:
+        for card in trello_list['cards']:
+            item = Item.from_trello_card(card, trello_list)
+            cards.append(item)
+
+    print(cards)
```

Comment:
This print statement is really helpful for debugging but it's worth noting that leaving it in is a recipe for potentially risky data leaks - it's easy to imagine users putting sensitive info into their to-dos ("go to the doctor about <condition>", "email <address> about <sensitive business matter>") and this sort of print statement slipping into production could easily mean that we have to treat our logs as containing sensitive personal data which isn't ideal, so a good habit to remove this now